### PR TITLE
LR arrow fix/ Scrolling on org for bounties

### PR DESCRIPTION
**Pull request by Vayras**
## Describe your changes
The left and right arrows, which are used to scroll between org bounties were not working for second brain and other org's as well. After testing I have changed the api call in `main.ts`, not sending any query params worked 

## Loom
https://www.loom.com/share/3ab1c505697b460f9698612287becb95

## Issue ticket number and link
issue: #333

## Type of change
UX change

Please delete options that are not relevant.
- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have tested on Chrome and Firefox
- [x] I have provided a screenshot or recording of changes in my PR if there were updates to the frontend

### CODE REVIEW

**Comment by ecurrencyhodler** [comment]
Thanks for addressing the no delete page and also ordering of bounties while scrolling. 

Can you also address this piece here?
` If there is no bounty, then no action should occur. Also the arrow to click the next button should be hidden.`

**Comment by Vayras** [comment]
@ecurrencyhodler sure will do

File: src/store/main.ts [hunk 1]
```diff
@@ -1877,58 +1877,19 @@ export class MainStore {
...
 
-  async getOrganizationPreviousBountyByCreated(
-    org_uuid: string,
-    bountyId: string
-  ): Promise<number> {
+  async getOrganizationPreviousBountyByCreated(org_uuid: string, created: number): Promise<number> {
     try {
-      const params = { languages: this.bountyLanguages, ...this.orgBountiesStatus };
-
-      const queryParams: QueryParams = {
```
Comment (elraphty) [inline src/store/main.ts:1915]: This won't fix it @Vayras, the issue is when the left and right arrow is clicked the Status goes back to the default state and all the statuses become false again, hence the wrong data.

**Comment by Vayras** [comment]
@elraphty I have addressed the changes requested
https://www.loom.com/share/5fa1976359a949c4b5986917998966ff

**Comment by ecurrencyhodler** [comment]
Video LGTM. Great work.

**Comment by elraphty** [comment]
@Vayras  https://www.loom.com/share/98e1c714187a451dbb6036b7d95af8bd, ensure the filter status used to fetch the bounties is what is sent to the next and previous bounty calls.

**Comment by Vayras** [comment]
> @Vayras  https://www.loom.com/share/98e1c714187a451dbb6036b7d95af8bd, ensure the filter status used to fetch the bounties is what is sent to the next and previous bounty calls.

@elraphty looks like a problem with status not resetting properly I'll fix this quick

**Comment by Vayras** [comment]
@elraphty I have added the commit that addresses your review

**Comment by elraphty** [comment]
> @elraphty I have added the commit that addresses your review

@Vayras please try to test with incognito, the error persists 
https://www.loom.com/share/b322dc2fd4214bfcb45cbfdb5fec26e0
also, try making the API-only request with Postman to get some more understanding

**Comment by Vayras** [comment]
@elraphty absolutely, not sure why this is not working on the other two browsers, if you need me to make any changes let me know, Thanks

**Comment by elraphty** [comment]
@Vayras tested again locally, it does go to the right next page when paid status is selected 
https://www.loom.com/share/da606fec0f8c40848489b81b49fee1c1

**Comment by Vayras** [comment]
> @Vayras tested again locally, it does go to the right next page when paid status is selected 
> https://www.loom.com/share/da606fec0f8c40848489b81b49fee1c1

That's great, it's working like it's expected 🚀

**Comment by elraphty** [comment]
> > @Vayras tested again locally, it does not go to the right next page when the paid status is selected
> > https://www.loom.com/share/da606fec0f8c40848489b81b49fee1c1
> 
> That's great, it's working like it's expected 🚀

No, it's not, when paid bounty status is selected the order breaks .

**Comment by Vayras** [comment]
Okay did not notice that, let me check and get back with a fix

**Comment by Vayras** [comment]
@elraphty I could not reproduce the same on my end it work as intended as shown in the loom video
https://loom.com/share/bc2962c8386f40f9b60d4c868b6a7aaf

**Comment by elraphty** [comment]
@kevkevinpal can you please help me test this PR locally, I have tried several on my computer, even on Incognito mood, it doesn't seem to work for me, but it works seamlessly on @Vayras end.

**Comment by kevkevinpal** [comment]
> @kevkevinpal can you please help me test this PR locally, I have tried several on my computer, even on Incognito mood, it doesn't seem to work for me, but it works seamlessly on @Vayras end.

Yea the PR seems to be working fine for me are you on this commit when testing?
`324b9b4cef9fc176f1dd4e089f312285c31a2bb7`

You can check by doing `git log -1`

**Comment by elraphty** [comment]
> git log -1

Yeah I'm on  324b9b4cef9fc176f1dd4e089f312285c31a2bb7

**Comment by elraphty** [comment]
> > git log -1
> 
> Yeah I'm on [324b9b4](https://github.com/stakwork/sphinx-tribes-frontend/commit/324b9b4cef9fc176f1dd4e089f312285c31a2bb7)

**Pull request merged**